tests/agent-templates.patch.test.ts

```
@@ -0,0 +1,501 @@
+  beforeAll,
+  beforeEach,
+  describe,
+  expect,
+  test,
+} from "bun:test";
+import express from "express";
+import { agentTemplatesRouter } from "@/api/v2/agent-templates/agent-templates.router";
+import { jsonMiddleware } from "@/middleware/json";
+import { noRouteMiddleware } from "@/middleware/noRoute";
+import { pinoMiddleware } from "@/middleware/pino";
+import { createJwtToken } from "@/utils/jwt";
```

ESLint

[error] Replace `createJwtToken·}·from·"@/utils/jwt` with `ADMIN_ACCOUNT_ID·}·from·"@/utils/constants`

(prettier/prettier)